app/assets/javascripts/components/form/form_paragraph.es6.jsx

```diff
@@ -0,0 +1,60 @@
+class FormParagraph extends Component {
+
+  static get propTypes() {
+    return {
+      title: React.PropTypes.string.isRequired,
+      placeHolder: React.PropTypes.string,
+      value: React.PropTypes.string,
+    };
+  }
+
+  static get defaultProps() {
+    return {};
+  }
+
+  handleChange(event) {
+   var node = React.findDOMNode(this.refs.container);
+
+   console.log(node.value);
+  }
+
+  componentDidMount() {
+    var node = React.findDOMNode(this.refs.container);
```

Comment:
Let's use `ReactDOM.findDOMNode` instead of `React.findDOMNode` since the latter is deprecated!